tests/system/test_agent_keep_alive/data/messages.yml

```diff
@@ -0,0 +1,61 @@
+---
+wazuh-manager:
+  - regex: ".*Checking for keys file changes."
+    path: "/var/ossec/logs/ossec.log"
+    timeout: 20
+  - regex: ".*Updating state file."
+    path: "/var/ossec/logs/ossec.log"
+    timeout: 20
+  - regex: ".*Updating shared files sums."
+    path: "/var/ossec/logs/ossec.log"
+    timeout: 20
+  - regex: ".*End updating shared files sums."
+    path: "/var/ossec/logs/ossec.log"
+    timeout: 20
+  - regex: ".*Updating state file."
+    path: "/var/ossec/logs/ossec.log"
+    timeout: 20
+  - regex: ".*inserting.*"
+    path: "/var/ossec/logs/ossec.log"
+    timeout: 20
+  - regex: ".*merged.mg*"
+    path: "/var/ossec/logs/ossec.log"
+    timeout: 20
+  - regex: ".*agent_ip.*"
+    path: "/var/ossec/logs/ossec.log"
+    timeout: 20
+  - regex: ".*reading.*"
+    path: "/var/ossec/logs/ossec.log"
+    timeout: 20
+  - regex: ".*merged.mg*"
+    path: "/var/ossec/logs/ossec.log"
+    timeout: 20
+  - regex: ".*agent_ip.*"
+    path: "/var/ossec/logs/ossec.log"
+    timeout: 20
+  - regex: ".*reading.*"
+    path: "/var/ossec/logs/ossec.log"
+    timeout: 20
```

Comment:
Using the default configuration for HostMonitor ( `update_position` `False` value for QueueMonitor), this messages list will not ensure that multiple logs `.*merged.mg*`, `.*agent_ip.*`, `.*reading.*`  appears in master log.
Also, these messages are maybe too generic and we should consider improving them

In addition, escape `.` final character in order to be more precise in the regex.